docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,14 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora/CentOS/RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
+then even after following the registry xref:../openshift/openshift-docker-registry.adoc#[documents], 
+you might see an authorization error when trying to push an image to exposed registry 
+because of incompatibility between the docker client shipped with Fedora/CentOS/RHEL and docker daemon which is running inside these ISO.
+
+Workaround: Start minishift using CentOS xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] `minishift start --iso-url centos`
```

Comment:
Workaround: Use the CentOS ISO xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] , when using Minishift on Fedora, CentOS or RHEL. For example, 

`minishift start --iso-url centos`